packages/react-components/src/atoms/TagLabel.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import css from '@emotion/css';
+import { steel, lead } from '../colors';
+
+const borderWidth = 1;
+const containerStyles = css({
+  display: 'flex',
+  cursor: 'default',
+  justifyContent: 'center',
+  alignItems: 'center',
+
+  marginTop: '6px',
+  marginBottom: '6px',
+});
+
+const styles = css({
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+
+  borderStyle: 'solid',
+  borderWidth: `${borderWidth}px`,
+  borderColor: steel.rgb,
+  borderRadius: '6px',
+
+  transition: '200ms',
+  paddingLeft: '6px',
+  paddingRight: '6px',
+
+  color: lead.rgb,
+  fontSize: '0.8em',
+});
+
+type TagLabelProps = {
+  readonly highlight?: boolean;
```

Comment:
remove

Reply:
Can remove but I'm following https://3.basecamp.com/4011604/buckets/16601866/uploads/2894021447.

Reply:
I know and it can be reintroduced, but it seems dead / not implemented atm and users of the component might expect  something to happen when settings the prop